Firebase/InAppMessaging/Data/FIRIAMMessageDefinition.m

```diff
@@ -60,6 +60,15 @@ - (BOOL)messageHasExpired {
   return self.endTime < [[NSDate date] timeIntervalSince1970];
 }
 
+- (BOOL)messageRenderedOnAppLaunchEvent {
+  for (FIRIAMDisplayTriggerDefinition *nextTrigger in self.renderTriggers) {
+    if (nextTrigger.triggerType == FIRIAMRenderTriggerOnAppLaunch) {
```

Comment:
could we return this evaluation vs return YES and return no? ie: return nextTrigger.triggerType == Blah?

Reply:
Not in this case, since there can be multiple triggers. We have to iterate and see if any of the triggers are `APP_LAUNCH`.